### PR TITLE
Added a new "checks" property

### PR DESCRIPTION
**Pull request by huntc**
The checks property permits the easy declaration of a component that uses ConductR's inbuilt check command in order to signal ConductR when a bundle component is ready.

### CODE REVIEW

File: src/sbt-test/sbt-bundle/checks/project/build.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+sbt.version=0.13.7
```
Comment (hseeberger) [inline src/sbt-test/sbt-bundle/checks/project/build.properties]: s/7/8

File: src/sbt-test/sbt-bundle/checks/test [hunk 2]
```diff
@@ -0,0 +1,3 @@
+> bundle:dist
+
+> checkBundleConf
```
Comment (hseeberger) [inline src/sbt-test/sbt-bundle/checks/test]: Trailing new line, please.

**Comment by hseeberger** [comment]
Nice. LGTM, apart from two comments.

**Comment by eed3si9n** [comment]
Cool. Thanks for this!

**Pull request merged**